docs/project/doc-style.md

```diff
@@ -93,9 +93,9 @@ More info from the always handy [Grammar Girl site](http://www.quickanddirtytips
 
 ### Pronouns
 
-It's okay to use first and second person pronouns. Specifically, use "we" to
+It's okay to use first and second person pronouns, especially if it lets you avoid a passive construction. Specifically, always use "we" to
 refer to Docker and "you" to refer to the user. For example, "We built the
-`exec` command so you can resize a TTY session."
+`exec` command so you can resize a TTY session." That said, in general, try to write simple, imperative sentences that avoid the use of pronouns altogether. Say "Now, enter your SSH key" rather than "You can now enter your SSH key."
```

Comment:
Actually... (nit) I think it's missing a full stop after "TTY session."

(and not wrapped to 80-chars)